py/autoevals/test_llm.py

```diff
@@ -14,7 +16,15 @@ def test_template_html():
     assert chevron.render(template_triple, dict(output="Template<Foo>")) == "Template<Foo>"
 
 
-def test_openai():
+@pytest.fixture
```

Comment:
This isn't as useful IMO as mocking and checking that the proper client is getting called, but better than nothing and follows the current pattern for the suite.